content/workshops/rhel_8/exercise1.1.adoc

```diff
@@ -0,0 +1,137 @@
+---
+title: "Exercise 1.1 - Managing Updates Using the Web Console"
+workshops: rhel_8
+workshop_weight: 11
+layout: lab
+---
+
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+:figure-caption!:
+:become_url: http://docs.ansible.com/ansible/become.html
+:dir_url: http://docs.ansible.com/ansible/playbooks_best_practices.html
+:package_url: http://docs.ansible.com/ansible/package_module.html
+:service_url: http://docs.ansible.com/ansible/service_module.html
+:yaml_url: http://docs.ansible.com/ansible/YAMLSyntax.html
+
+
+= Exercise 1.1 - Managing Updates Using the Web Console
+
+== Exercise Description
+
+Now that you've toured the web console, lets work through updating the system. 
+
+== Section 1: Basic Updating
+
+As you did in the previous section, navigate to the `Software Updates` section of the Web Console, by clicking on the `Software Updates` link, in the left-hand menu. There will then be a brief interval where `Refreshing package information` will appear, along with a progress bar. Once that completes, you can proceed with the exercise.
```

Comment:
Line 26: 1) “Web Console” has been lower case to this point. 2) Change “by clicking on the” to “by clicking the”.  3) Remove the comma after ‘Software Updates’ link. 4) A ‘Refreshing package information’ message will briefly display with a progress bar. Upon completions of the refresh, you may proceed with the exercise.